examples/connector/python/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+python3 -m venv connector_run
```

Comment:
Let's display some information for the steps so the user knows what's happening..